docs/Update-MarkdownHelp.md

```diff
@@ -1,180 +1,195 @@
-Specify a **System.Text.Encoding** object.
-For more information, see [Character Encoding in the .NET Framework](https://msdn.microsoft.com/en-us/library/ms404377.aspx) in the Microsoft Developer Network.
-For example, you can control Byte Order Mark (BOM) preferences.
-For more information, see [Using PowerShell to write a file in UTF-8 without the BOM](http://stackoverflow.com/questions/5596982/using-powershell-to-write-a-file-in-utf-8-without-the-bom) at the Stack Overflow community.
-
-
-```yaml
-Type: Encoding
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: 1
-Default value: UTF8 without BOM
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
-### -LogAppend
-Indicates that this cmdlet appends information to the log instead overwriting it.
-
-
-```yaml
-Type: SwitchParameter
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
-### -LogPath
-Specifies a file path for log information.
-The cmdlet writes the VERBOSE stream to the log.
-If you specify the *Verbose* parameter, this cmdlet also writes that information to the console. 
-
-
-```yaml
-Type: String
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: 2
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
-### -Path
-Specifies an array of paths of markdown files and folders to update.
-
-
-```yaml
-Type: String[]
-Parameter Sets: (All)
-Aliases: 
-
-Required: True
-Position: 0
-Default value: None
-Accept pipeline input: True (ByValue)
-Accept wildcard characters: False
-```
-
-### -AlphabeticParamsOrder
-Order parameters alphabetically by name in PARAMETERS section.
-There are 2 exceptions: -Confirm and -WhatIf parameters will be the last. 
-These parameters are common and hence have well-defined behavior.
-
-```yaml
-Type: SwitchParameter
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
-### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
-
-## INPUTS
-
-### String[]
-You can pipe an array of paths to this cmdlet.
-
-## OUTPUTS
-
-### System.IO.FileInfo[]
-This cmdlet returns a **FileInfo[]** object for updated files.
-
-## NOTES
-
-## RELATED LINKS
-
-[Character Encoding in the .NET Framework](https://msdn.microsoft.com/en-us/library/ms404377.aspx)
-
-[Using PowerShell to write a file in UTF-8 without the BOM](http://stackoverflow.com/questions/5596982/using-powershell-to-write-a-file-in-utf-8-without-the-bom)
-
+---
+external help file: platyPS-help.xml

```

Comment:
please, set you git preferences to "Checkout Windows-style, commit Unix-style line endings"

```
git config --global core.autocrlf true
```

That will remove this `^M` signs which are artifacts of the `\r` in `\r\n` windows line endings

Also the fact that entire file shows up as a diff is alarming, but it probably relates to the line ends as well.

For more details see
https://help.github.com/articles/dealing-with-line-endings/